internal/host/host.go

```diff
@@ -62,6 +62,11 @@ var InstallationProgressTimeout = map[models.HostStage]time.Duration{
 	"DEFAULT":                                   60 * time.Minute,
 }
 
+var IgnoreTimeoutCheckOnStatuses = [...]string{
+	models.HostStatusInstallingPendingUserAction,
```

Comment:
What is the difference between the two?

Reply:
HostStatusInstallingPendingUserAction - wrong boot order and we need the user to boot that specific host from disk.
HostStatusResettingPendingUserAction - post cluster reset we need the user to reboot all nodes back to discovery mode.

Reply:
OK thanks, anyway this fix is not relevant because we don't have timeout for those states anyway

Reply:
As I see it. if the cluster has a node in one of these states, installation timeout should be off. 
So while this is under host.go it cares about other hosts in the cluster.